Makefile

```diff
@@ -0,0 +1,5 @@
+install:
+    poetry install
+
+brain-games:
+    poetry brain-games
```

Comment:
where is this script? it should be written in the project `pyproject.toml` file. btw, have you tried to run it? I'm sure it'll fail